01-introduction.Rmd

```diff
@@ -17,7 +17,7 @@ A major aim of this book is to make geographical data analysis more accessible.
 R is a flexible language that allows access to many spatial software libraries (see section \@ref(why-geocomputation-with-r)).
 Before going into the details of the software, however, it is worth taking a step back and thinking about what we mean by geocomputation.
 
-## What is geocomputation?
+## A definition of geocomputation, and why it is in the book's title
```

Comment:
Simple and concise - yes please! (We can always change to something more complex and potentially accurate later.)